src/app/components/search-bar/search-bar.component.html

```
@@ -1,18 +1,18 @@
-<div>
+<div class="toolbar-placeholder">
   <mat-toolbar>
     <div class="content">
-      <button mat-flat-button aria-label="back" class="button backButton" (click)="goBack()">
+      <button mat-flat-button aria-label="back" class="button actionButton backButton" (click)="goBack()">
         <mat-icon style="color: #543D27;">arrow_back</mat-icon>
       </button>
       <input #searchInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)